packages/one-app-bundler/utils/time.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+const chalk = require('chalk');
+const { performance } = require('perf_hooks');
+
+function printTimeDuration(name, duration) {
+  console.log('\n[one-app-bundler]: %s took %s seconds to complete.\n', chalk.bold.green(name), chalk.bold.green(`${duration / 1000}`));
```

Comment:
should we change this too `The build took 100 seconds to complete` would seem more generic

Reply:
Agree with @nellyk, we should round the seconds to 2 decimal places too. Another option is 

`Module Bundle build done in 7.22 seconds` whatever feels more natural and short, people will see that message a lot! 😅

Reply:
Agree with this, will update

Reply:
updated (84f0037)